### DISCUSSION TITLE
Are SRV records supported?

### DISCUSSION THREAD

**Discussion by thrashwerk** [ceph/ceph-csi#3697] [Q&A]
Does Ceph-CSI support SRV records instead of specifying all monitors?

https://docs.ceph.com/en/quincy/rados/configuration/mon-lookup-dns/

I've tried not specifying monitors at all since clients should (I think) be able to get the records through DNS but that didn't work. I got errors Ceph-CSI could determine the monitors
```
failed to fetch monitor list using clusterID (96df99f6): empty monitor list for cluster ID (96df99f6) in config
```

Setting the SRV record as a monitor didn't seem to work either.

**Comment by SeanWallace** [comment]
No, they don't seem to be fundamentally due to the way that Monitor lookups are performed. I have opened a feature request to support this: https://github.com/ceph/ceph-csi/issues/5102
